.github/ISSUE_TEMPLATE/categorize_tracker.yml

```diff
@@ -64,19 +64,11 @@ body:
     validations:
       required: false
  
-  - type: textarea
-    id: tracker-name
-    attributes:
-      label: Suggested Name
-      description: Suggest a name for this unidentified tracker.
-      placeholder: CNN Analytics
-    validations:
-      required: false
   - type: dropdown
     id: tracker-category
     attributes:
       label: Category
-      description: Which category does this request **BEST** belong to? You can find category descriptions [here](https://example.com).
+      description: Which category does this request **BEST** belong to? You can find category descriptions here (forthcoming).

```

Comment:
If you need a link to the categories, this (unmerged) PR might be useful: https://github.com/ghostery/trackerdb/pull/42

It generates a site with all categories: https://github.com/ghostery/trackerdb/blob/264a8beb88eb1115a4109cd1c655a4c3d4f59a07/docs/categories.md

Reply:
I merged the PR now. You can link to the current page, as done here: https://github.com/ghostery/trackerdb/commit/c9f37a44cb714b91eb3e5bbe013280e21d730ddf

Maybe that here will work: `../../docs/categories.md`

Reply:
done